cmd/docker-machine-driver-kvm/Makefile

```diff
@@ -1,4 +1,4 @@
 default: build
 
 build:
-	GOGC=off go build -i -o docker-machine-driver-kvm
+	GOGC=off go build -tags libvirt.1.2.14 -i -o docker-machine-driver-kvm
```

Comment:
I've been meaning to improve the build a bit... and this helped motivate me to get some extra build support in place to generate binaries for various host systems. https://github.com/dhiltgen/docker-machine-kvm/commit/f328c4b81d36269845812a6bd0046e64a9c7d877

I think we should be able to at least partially answer the compatibility question on this PR by seeing if it builds on ubuntu14.04.  Then we could try to do a create in a container, which should fail due to the missing libvirt socket, but as long as it errors that way and not with some obscure C lib link problem, then I think we're good.